tnt/src/main/java/org/dcsa/conformance/standards/tnt/party/TntPublisher.java

```diff
@@ -211,18 +191,16 @@ public static ArrayNode sortJsonArray(ArrayNode jsonArray, List<SortCriteria> cr
     List<JsonNode> jsonNodeList = new ArrayList<>();
     jsonArray.forEach(jsonNodeList::add);
 
-    jsonNodeList.sort(
-        (node1, node2) -> {
-          int comparisonResult = 0;
-          for (SortCriteria criterion : criteria) {
-            comparisonResult =
-                compareNodesByField(node1, node2, criterion.field(), criterion.direction());
-            if (comparisonResult != 0) {
-              break; // Stop comparing if a difference is found
-            }
-          }
-          return comparisonResult;
-        });
+    jsonNodeList.sort((node1, node2) -> {
+      int comparisonResult = 0;
+      for (SortCriteria criterion : criteria) {
+        comparisonResult = compareNodesByField(node1, node2, criterion.field(), criterion.direction());
+        if (comparisonResult != 0) {
+          break; // Stop comparing if a difference is found
```

Comment:
Does this also work? Its intentions are clear already. 
```suggestion
          return comparisonResult;
```